ouroboros-network/test/Test/Ouroboros/Network/BlockFetch.hs

```diff
@@ -52,6 +52,7 @@ import           Ouroboros.Network.Mock.ConcreteBlock
 import           Ouroboros.Network.NodeToNode.Version (isPipeliningEnabled)
 import           Ouroboros.Network.Protocol.BlockFetch.Type (BlockFetch)
 
+import           Ouroboros.Network.NodeToNode (ConnectionId (..))
```

Comment:
The block fetch should be using remote peer address not `ConnectionId`.   I know this is just testing, but it gives an example how to set things - so I'd prefer to preserve the original if possible.